Monika After Story/game/script-story-events.rpy

```diff
@@ -534,6 +610,8 @@ label mas_crashed_long_uthere:
     label .end:
         m "Anyway..."
         m 1eua "What should we do today?"
+        $ mas_apology_reason = "the game crashing"
```

Comment:
all crashed flows will eventually reach `mas_crashed_post`, so you should just do apology setting in there.